### PR TITLE
Redirect GitHub link to proper repository. 

### PR DESCRIPTION
**Pull request by Matthieu-ui**
Link on landing page was for the archived repository!

### CODE REVIEW

**Review by Matthieu-ui — COMMENTED** [review]
fixed three links

**Review by devarshishimpi — CHANGES_REQUESTED** [review]
Could you add the blog link back?

File: index.html [hunk 1]
```diff
@@ -40,15 +40,16 @@
...
                 <a href="./contact.html" >Contact</a>
             </div>
         </div>
         <div id="mobilenav">
             <a href="#feature">Features</a>
-            <a href="https://github.com/devarshishimpi/Stone-CSS-Website" target="_blank">Github</a>
-            <a href="https://blog.stonecss.com" target="_blank">Blog</a>
+
+            <a href="https://github.com/devarshishimpi/stonecss" target="_blank">Github</a>
+
```
Comment (MrKrishnaAgarwal) [inline index.html:52]:
```suggestion
 <a href="https://blog.stonecss.com" target="_blank">Blog</a>
```

Please add these changes back.

File: index.html [hunk 2]
```diff
@@ -89,8 +90,7 @@ <h6>Make a stunning website in minutes by using the Stone CSS framework</h6>
                 <hr>
                 <div id="right-footer">
                     <a href="#feature">Features</a>
-                    <a href="https://github.com/devarshishimpi/Stone-CSS-Website" target="_blank">Github</a>
-                    <a href="https://blog.stonecss.com" target="_blank">Blog</a>
+                    <a href="https://github.com/devarshishimpi/stonecss" target="_blank">Github</a>
```
Comment (MrKrishnaAgarwal) [inline index.html:93]:
```suggestion
                    <a href="https://github.com/devarshishimpi/stonecss" target="_blank">Github</a>
                    <a href="https://blog.stonecss.com" target="_blank">Blog</a>
```
Please add the blog link here back too.

**Review by MrKrishnaAgarwal — CHANGES_REQUESTED** [review]
@Matthieu-ui Please make the requested changes as mentioned below in comments.

**Review by MrKrishnaAgarwal — APPROVED** [review]
Looks good now!

**Review by devarshishimpi — APPROVED** [review]

**Pull request merged**